c2me-threading-chunkio/src/main/java/com/ishland/c2me/threading/chunkio/mixin/MixinThreadedAnvilChunkStorage.java

```diff
@@ -68,7 +68,7 @@ public MixinThreadedAnvilChunkStorage(Path path, DataFixer dataFixer, boolean bl
 
     @Shadow
     @Final
-    private ServerWorld world;
+    ServerWorld world;
```

Comment:
Is this change strictly necessary?

Reply:
Oh whoops, this one wasn't needed, although it does show up in my ide as a warning for some reason